coil-base/src/main/java/coil/memory/BitmapReferenceCounter.kt

```diff
@@ -33,6 +31,7 @@ internal class BitmapReferenceCounter(
     /**
      * Increase the reference count for this [Bitmap] by one.
      */
+    @Synchronized
     fun increment(bitmap: Bitmap) {
```

Comment:
This PR also makes `StrongMemoryCache`, `WeakMemoryCache`, and `BitmapReferenceCounter` thread safe (so `MemoryCache` is thread safe. This also also required for another feature coming later!